sos/plugins/kdump.py

```diff
@@ -45,4 +46,25 @@ def setup(self):
             "/etc/default/kdump-tools"
         ])
 
+
+class CosKDump(KDump, CosPlugin):
+
+    option_list = [
+        ("all_dumps", "enable capture for all kernel dumps", "", False),
+        ("latest_dump", "enable capture for latest kernel crash dump",
+            "", False),
+    ]
+
+    def setup(self):
+        super(CosKDump, self).setup()
+        self.add_cmd_output('ls -alRh /var/kdump*')
+        if self.get_option("all_dumps"):
+            self.add_copy_spec([
+                "/var/kdump-*"
```

Comment:
Assuming `/var/kdump` is the current dump and `/var/kdump-*` are elder ones, shouldnt we collect here `/var/kdump*`?

In the current implementation, `all_dumps` will skip collection of the current dump.